### ISSUE TITLE
JAWS accessibility for Field component 

### ISSUE THREAD

**Issue by cliu02** [bloom-housing/ui-components#136]
This issue is from a DAHLIA accessibility audit. 

**Description**: When a JAWS user accesses the Field component to enter their lottery number, the screen reader skips the field. The screen reader does not announce "Enter your lottery number" as expected. This issue maps to [WCAG 3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html): "Labels or instructions are provided when content requires user input."

Part 2 of [Issue #123](https://github.com/bloom-housing/ui-components/issues/123) was previously completed for this issue with recommended changes, but these changes did not resolve the issue. 

**How should this work?** 
1. Go to https://dahlia-full.herokuapp.com/listings/a0W4U00000KnVhoUAF?react=true 
2. Click on View Lottery Results
![image](https://github.com/bloom-housing/ui-components/assets/32694266/d3116b93-562c-48bd-bdbc-99863f80393d)
3. For the input, JAWS should read the title "Enter Your Lottery Number".
![image](https://github.com/bloom-housing/ui-components/assets/32694266/1e138602-94e3-49a7-ba8b-cbac5d1f07ab)

**Comment by ColinBuyck** [comment]
For this... I don't have too much to add or valuable insight but would just really love the ability to test with JAWS. Are there any updates on getting an Assistiv Labs account with JAWS?
